src/components/UpcomingEventsList.tsx

```diff
@@ -164,32 +139,19 @@ const UpcomingEventsList = () => {
                     />
                   )
                 )}
-              </Grid>
-            </Box>
+              </div>
+            </div>
           )
         })}
 
-      <Box
```

Comment:
Did this do something or just a mistake?

Reply:
Hmm good question xD I don't think this is doing anything visually, its adding just a bit of space...I think it is safe to remove it